curp-external-api/src/log_index.rs

```diff
@@ -0,0 +1,2 @@
+/// Log Index
+pub type LogIndex = u64;
```

Comment:
I recommend define this type in other file and remove this file

Reply:
You mean move it to the `lib.rs` file?

Reply:
Yes